lib/DimmableLightDevice.js

```diff
@@ -78,9 +87,10 @@ module.exports = {
 				this.node.endpoints[0].clusters.genLevelCtrl.read('currentLevel')
 					.then(currentDimValue => {
 						this.log('currentLevel: ', currentDimValue);
-						this.parsedcurrentDimvalue = Math.round((currentDimValue / 254) * 100) / 100;
-						this.log('currentdimvalue: ', this.parsedcurrentDimvalue);
-						this.setCapabilityValue('dim', this.parsedcurrentDimvalue);
+						const parsedcurrentDimvalue = Math.round((currentDimValue / 254) * 100) / 100;
+						this.log('currentdimvalue: ', parsedcurrentDimvalue);
+						this.setCapabilityValue('dim', parsedcurrentDimvalue);
+						if (parsedcurrentDimvalue === 0) this.setCapabilityValue('onoff', false);
```

Comment:
Moved this check to the interval so `parsedcurrentDimvalue` does not have to be a global variable and the dim value is checked every time instead only after the last interval. I expect it could happen that the dim value is 0 before the timeout ends when dimming from 0.01 to 0.00 with an interval of 30 minutes